editions/tw5.com/tiddlers/widgets/EditTextWidget.tid

```diff
@@ -20,7 +20,7 @@ The content of the `<$edit-text>` widget is ignored.
 |field |The field to edit (defaults to `text`). Takes precedence over the `index` attribute |
 |index |The index to edit |
 |default |The default text to be provided when the target tiddler doesn't exist |
-|class |A CSS class to be assigned to the generated HTML editing element |
+|class |A CSS class to be assigned to the generated HTML editing element. To fully expand the widget horizontally, use `tc-max-width` |
```

Comment:
@saqimtiaz thanks. To not confuse the reader by bringing up DOM etc, would this suffice "To display the widget fully expanded horizontally, use..."

Reply:
@saqimtiaz would it work to simply refer to it as the "element", rather than "HTML element"? Other than brevity, my ambition is to not introduce superfluous terms that the reader can get hung up on. (*"HTML? Isn't this wikitext?"* - etc)

Reply:
@twMat the term "HTML editing element" is already used in the previous sentence, so consistency with that seems logical.